### PR TITLE
Release/1.3.1

### PR DESCRIPTION
**Pull request by missedone**

### CODE REVIEW

File: remote-test/src/test/groovy/Tester.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @Grab(group = 'org.osgi', module = 'osgi.core', version = '6.0.0')
-@Grab(group = 'org.testng.testng-remote', module = 'testng-remote', version = '1.3.1-SNAPSHOT')
+@Grab(group = 'org.testng.testng-remote', module = 'testng-remote', version = '1.3.1')
```
Comment (juherr) [inline remote-test/src/test/groovy/Tester.groovy]: Why not `1.4.0-SNAPSHOT`?

Comment (missedone) [reply remote-test/src/test/groovy/Tester.groovy]: 1.4.0-SNAPSHOT is not yet available immediately after the mvn release

Comment (juherr) [reply remote-test/src/test/groovy/Tester.groovy]: ok

File: remote-test/src/test/groovy/Tester.groovy [hunk 2]
```diff
@@ -1,5 +1,5 @@
 @Grab(group = 'org.osgi', module = 'osgi.core', version = '6.0.0')
-@Grab(group = 'org.testng.testng-remote', module = 'testng-remote', version = '1.3.1-SNAPSHOT')
+@Grab(group = 'org.testng.testng-remote', module = 'testng-remote', version = '1.4.0-SNAPSHOT')
```
Comment (cbeust) [inline remote-test/src/test/groovy/Tester.groovy]: Maybe extract a constant since this version is used again below?

Comment (missedone) [reply remote-test/src/test/groovy/Tester.groovy]: `@Grab` is sort of special as it's the first pass in the lifecycle, neither var nor constant is supportted as i remember.
will see if anyway out later

Comment (cbeust) [reply remote-test/src/test/groovy/Tester.groovy]: I see, thanks.

**Pull request merged**